File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntActionExecuteWorkflow.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdAutomation;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioEntActionExecuteWorkflow extends StudioEntAction
{
  @Nullable
  public MetaIdAutomation automationId;

  @Nullable
  public EntVdWorkflowPointer workflowPointer;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntActionExecuteWorkflow.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdWorkflowPointer
import com.neome.api.meta.base.Types.MetaIdAutomation
import com.neome.api.meta.base.dto.StudioEntAction

interface StudioEntActionExecuteWorkflow : StudioEntAction
{
  val automationId: MetaIdAutomation?
  val workflowPointer: EntVdWorkflowPointer?
}